core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1072,6 +1072,12 @@ public static long bitAnd(long b0, long b1) {
     return b0 & b1;
   }
 
+  // ~
```

Comment:
is `~` supported in Parser, or is this `~` just a comment without indicating that parser supports `~`?

Reply:
> supported in Parser

`~` Not supported in Parser.